src/util.ts

```diff
@@ -397,7 +397,7 @@ export function isInternalField(name: string) {
  * Normalize angle to be within [0,360).
  */
 export function normalizeAngle(angle: number) {
-  return ((angle % 360) + 360) % 360;
+  return angle !== undefined ? ((angle % 360) + 360) % 360 : undefined;
```

Comment:
I think normal ifs make code more understandable. Ternaries make sense when you assign to a variable. 

```suggestion
  if (angle === undefined) {
  	return undefined;
  }
  return ((angle % 360) + 360) % 360;
```